.github/workflows/pull-request-token.yml

```diff
@@ -67,6 +67,14 @@ jobs:
           ./token/twoxtx-setup.sh
           ./ci/cargo-test-bpf.sh token/program-2022-test
 
+      - name: Build and test token-2022 with "serde" activated
+        run: |
+          cargo +"$RUST_STABLE" test-bpf \
+            --manifest-path=token/program-2022/Cargo.toml \
+            --features serde,serde_with \
```

Comment:
@joncinque this looks suspicious—i had expected the `serde-feature = ["serde", "serde_with"]` in Cargo.toml to handle enabling this optional dependency, but `cargo` kept complaining to me whenever i omitted this feature. i have this here to make sure CI passes right now, but if you have some insight here that'd be useful!

<img width="549" alt="image" src="https://user-images.githubusercontent.com/1118534/177665168-c1ec7b7b-07b7-4d02-ad6f-bab8af499c03.png">